MoneyAPP/moneyapp/lib/main.dart

```diff
@@ -14,7 +14,8 @@ void main() {
   final apiSource = ApiDataSource(dio);
   runApp(
     BlocProvider(
-      create: (context) => TransactionsBloc(apiSource),
+      create: (context) =>
+          TransactionsCubit(apiSource), // Use TransactionsCubit
```

Comment:
Instead of calling constructor yourself, you have to utilise the other part of injectable, completing its purpose. Do this:
```dart
BlocProvider<TransactionsCubit>(
      create: (context) => getIt(),
```

What this will do is - `getIt()` will find the class it needs (in this case `TransactionsCubit`) and inject it by filling the constructor parameter with classes it already have in its scope. That means that you need to make sure that ApiSource is also in the scope, by marking it as a @singleton. Read what is the difference between singleton and injectable, and ask me if you need me to explain as well. If the code doesn't imply what class getIt needs to fill, you need to tell him by passing the class name as a generic parameter, like this `getIt<TransactionsCubit>()`. In this case, you could put it in getIt as generic parameter and omit putting it in BlocProvider as generic parameter as well. It is the same thing - compiler will adapt to whatever you do.